src/app/components/playback/playback_control.rs

```diff
@@ -149,6 +174,41 @@ impl PlaybackControl {
             .expect("error updating icon");
     }
 
+    fn update_shuffle(&self) {
+        let is_shuffled = self.model.state().playback.is_shuffled();
+        let playback_image = if is_shuffled && self.model.is_playing() {
+            "media-playlist-shuffle-symbolic"
+        } else {
+            "shuffle"
+        };
+        self.widget
+            .shuffle_button
+            .child()
+            .and_then(|child| child.downcast::<gtk::Image>().ok())
+            .map(|image| {
+                image.set_from_icon_name(Some(playback_image), image.icon_size());
+            })
+            .expect("error updating icon");
+    }
+
+
+    fn update_repeat(&self, mode: &RepeatMode) {
+        let playback_image = match mode {
+            RepeatMode::Song => "media-playlist-repeat-song-symbolic.symbolic",
+            RepeatMode::Playlist => "media-playlist-repeat-symbolic",
+            RepeatMode::None => "media-playlist-repeat",
```

Comment:
maybe `media-playlist-consecutive-symbolic`?

Reply:
(so that we have only symbolic icons ;) )

Reply:
Im not sure how it looks for you but that symbol looks very odd and out of place for me.
![Screenshot_20210731_123239](https://user-images.githubusercontent.com/63280150/127746405-00203f35-4a30-426b-b88c-88133f82aa1a.png)
I personally think making RepeatMode::None as non symbolic is better because it better shows that option is off.

Reply:
Can't see much on your screenshot but this would be what users with stock icons/themes would see:
![Capture d’écran de 2021-07-31 19-45-24](https://user-images.githubusercontent.com/6277494/127748299-b75c6266-285a-4b68-84b8-8954a33e32ec.png)

It's also the icon used by other media players, such as Lollypop

Reply:
Hmm The first two I get from using media-playlist-consecutive-symbolic.symbolic and media-playlist-repeat-song-symbolic.symbolic. If they look the same for the flathub pack (if they even work) should I leave it?